src/extract-exports/extract-exports.ts

```diff
@@ -7,6 +7,7 @@ import {extname} from 'node:path';
 export type ExportInfo = {
   name: string;
   exportKind?: 'type' | 'value' | null | undefined;
+  actualName?: string;
```

Comment:
Let’s rename it to exportName so that it matches the template variable ($exportName)
```typescript
exportName?: string | undefined;
```